scalyr_agent/builtin_monitors/apache_monitor.py

```diff
@@ -49,6 +49,7 @@
     "number, at which the Apache status module is served. The URL should end in `/?auto` to "
     "indicate that the machine-readable version of the page should be returned.",
     default="http://localhost/server-status/?auto",
+    insecure_http_url=True
```

Comment:
For consistency could we rename this as `allow_http`?

I know that could be confused with the unrelated allow_http for scalyr server but I think it's clearer to have the monitor config parameter with a similar name

Reply:
That sounds good. I wasn't sure about how to name it and insecure_http_url is definitelly confusing.
I will make allow_http=True by default and  allow_http=False will mark the option for https enforcing. 
Do you agree?

Reply:
Sounds good